### PR TITLE
Convert to new YAML syntax

### PR DESCRIPTION
**Pull request by tdutrion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,34 @@
...
-  resolves = ["PHP-CS-Fixer"]
-}
-
-action "PHP-CS-Fixer" {
-  uses = "docker://oskarstark/php-cs-fixer-ga"
-}
+on: push
+name: Main
+jobs:
+  php-cs-fixer:
```
Comment (OskarStark) [inline README.md]:
```suggestion
  PHP-CS-Fixer:
```

File: examples/.github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+on: push
+name: Main
+jobs:
+  pHP-CS-Fixer:
```
Comment (OskarStark) [inline examples/.github/workflows/push.yml]:
```suggestion
  PHP-CS-Fixer:
```

File: README.md (hunk 1 above)
Comment (tdutrion) [reply README.md]: The github migration tools was going for `pHP-CS-Fixer:` so I figured that key should be lower case at least for the first character, hence the lowercase.
Do you think we can leave it as is (lower)?

File: examples/.github/workflows/push.yml (hunk 2 above)
Comment (tdutrion) [reply examples/.github/workflows/push.yml]: Same as the other one

>The github migration tools was going for pHP-CS-Fixer: so I figured that key should be lower case at least for the first character, hence the lowercase.
>Do you think we can leave it as is (lower)?

I updated it to be all lowercase for now, let me know.

File: README.md (hunk 1 above)
Comment (OskarStark) [reply README.md]: > Do you think we can leave it as is (lower)?

I would say yes 👍

**Pull request merged**